src/mixture/4C_mixture_constituent_remodelfiber_lib.hpp

```diff
@@ -45,7 +45,7 @@ namespace Mixture
   [[nodiscard]] inline T get_exponential_fiber_strain_energy(
       const PAR::ExponentialFiberParameters& params, const T I4)
   {
-    if (I4 < 0 && !params.supports_compression_)
+    if (I4 < 1 && !params.supports_compression_)
       FOUR_C_THROW("The fiber is under compression, but does not support that.");
```

Comment:
```suggestion
    FOUR_C_ASSERT_ALWAYS(params.supports_compression_ || I4 >= 1, "The fiber is under compression, but does not support that.");
```
It's now also possible to write this in one line (but the current version is also fine).